submenu.js

```diff
@@ -0,0 +1,21 @@
+const submenu = document.getElementById("nav__hamburger-submenu-wrapper");
+const hamburgerMenu = document.getElementById("nav__hamburger-menu");
+const hamburgerIcon = document.getElementById("nav__hamburger-icon");
+
+export function forwardbutton() {
+    submenu.classList.toggle("--hidden");
+    hamburgerMenu.classList.toggle("--hidden");
+}
+
+export function submenuHide() {
+    if (!submenu.classList.contains("--hidden")) {
+        forwardbutton()
+    }
+}
+
+export function submenuControl() {
+    if (!submenu.classList.contains("--hidden")) {
+        submenu.classList.toggle("--hidden");
+        hamburgerIcon.classList.toggle("Close-icon");
+    }
+}
```

Comment:
What difference is between this function and the two functions above? Please describe in reply.